packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -57,6 +57,8 @@ import {
   ServiceOptions,
 } from '@loopback/core';
 
+/* eslint-enable @typescript-eslint/no-unused-vars */
```

Comment:
I am always reluctant to disable a linter rule for the entire file, because it encourages introducing even more violations of that rule in the future.

Can you please explain why is `no-unused-vars` triggered now? Is it possible to fix the violations or at least disable the rule only for specific lines?

Reply:
Same deal.

Reply:
And the same response :)

At the moment, I see the same `eslint-enable` comment twice, can you PTAL? I think the line 34 can be removed.